README.md

```diff
@@ -2,3 +2,13 @@
 This repository is used to document and define coding practices such as coding styles, writing commit messages and documentation.
 
 The code_conventions.md defines the general coding conventions, without going into language specifics. It is possible that certain languages have their own coding standards. If this is the case, an additional document is added, which provides addendums or changes to the general codestyle.
+
+Vocabulary
+---
+ Solution for hotend versus nozzle and {}_index versus {}_nr or {}_number name usage:
+
+ * Only use 'hotend'
```

Comment:
This is confusing. Only ever use "hotend"? Hotend hotend hotend hotend? Hotend hotend hotend, hotend hotend hotend Hotend hotend hotend hotend. Hotend hotend. Hotend hotend... Hotend!

Reply:
Hotend is the term for the hot part of the end of the extruder train. It should be used for that and for that only.

Reply:
It's just a bit confusing to read then. I'll suggest to choose one of the following:

* Remove the title `Vocabulary` and put the 3 dashes beneath the sentence below it. Make that the title.
* (My preference:) Specify what it's about within the bullet point. Such as:



    \* Use 'hotend' for the hot part at the end of the extruder train, never 'nozzle'.
    \* Only use 'nozzle' to refer to (attributes of) the Olsson block at the end of the hotend.

Reply:
Agreed.